test/model.test.js

```diff
@@ -707,7 +707,21 @@ describe.onServer('Remote Methods', function() {
       var callbackSpy = require('sinon').spy();
       var TestModel = app.models.TestModelForDisablingRemoteMethod;
       TestModel.on('remoteMethodDisabled', callbackSpy);
-      TestModel.disableRemoteMethod('findOne');
+      TestModel.disableRemoteMethod('findOne', true);
```

Comment:
In 2.x, the method `disableRemoteMethod` is not deprecated. Can we place keep this test unmodified to cover this API?